colossalai/zero/sharded_optim/sharded_optim_v2.py

```diff
@@ -130,8 +127,6 @@ def __init__(self,
                 f"After init ShardedOptimizerV2 consumes {self.get_memory_usage()[0] / 1e6} MB CUDA Memory!", ranks=[0])
 
         self._use_memory_tracer = self.model.use_memory_tracer
-        if self._use_memory_tracer:
-            GLOBAL_MODEL_DATA_TRACER.register_optimizer(self)
```

Comment:
I think we should keep this right now.
Since users may want to know the comsumption of their optimizers.

Reply:
Update `memory_usage` in next PR.

Reply:
ok